definitions/npm/bser_v1.x.x/flow_v0.22.x-/bser_v1.x.x.js

```diff
@@ -0,0 +1,58 @@
+declare type $npm$bser$BserTypes =
```

Comment:
Oh, it doesn't need to be I just copied this pattern from another def.
So I just move it inside the `declare module` as is? Or should I rename it to declare type `BserType`? But wouldn't any `declare type` inside a `declare module` define a named export?
I think that is why I initially put it on the top scope, as I thought prefixing it with $npm means it is private and does not count towards the named exports.